includes/class-receiver.php

```diff
@@ -501,10 +501,7 @@ public static function disable_wp_check_dupes( $dupe_id, $commentdata ) {
 			return $dupe_id;
 		}
 
-		if (
-			( isset( $commentdata['comment_type'] ) && 'webmention' === $commentdata['comment_type'] ) ||
-			( isset( $commentdata['comment_meta'] ) && ! empty( $commentdata['comment_meta']['semantic_linkbacks_type'] ) )
-		) {
+		if ( ! empty( $commentdata['comment_meta']['protocol'] ) && 'webmention' === $commentdata['comment_meta']['protocol'] ) {
```

Comment:
can we alway assume that the array-keys exist?

Reply:
This is a check that is only supposed to be triggered on a Webmention, but no issue with an existence check